routing/cross_mwm_graph.hpp

```diff
@@ -93,17 +93,13 @@ class CrossMwmGraph final
 
   void Clear();
 
-  // Applies |fn| to each nontransit transition. We never need to apply function to both transit
-  // and nontransit transitions.
-  // We may need to implement ForEachTransitTransition() or |isTransit| flag here when we'll have
-  // leaps for transit.
-  template <typename Fn>
-  void ForEachTransition(NumMwmId numMwmId, bool isEnter, Fn && fn)
+  // Returns nontransit transitions for mwm with id |numMwmId|.
```

Comment:
`/// \returns nontransit transitions for mwm with id |numMwmId|.` and below let's put down `\\\`

Reply:
done